lib/stdlib/src/math.erl

```diff
@@ -134,8 +134,31 @@ tan(_) ->
 tanh(_) ->
     erlang:nif_error(undef).
 
+-spec ceilf(X) -> float() when
+      X :: number().
+ceilf(_) ->
+    erlang:nif_error(undef).
+
+-spec floorf(X) -> float() when
+      X :: number().
+floorf(_) ->
+    erlang:nif_error(undef).
+
 %%% End of BIFs
 
+-spec ceil(X) -> integer() when
```

Comment:
It is more efficient to avoid the function call and just use an if as described at http://erlang.org/pipermail/erlang-questions/2009-March/042370.html